apps/src/lib/levelbuilder/script-editor/LessonDescriptions.jsx

```diff
@@ -93,11 +94,14 @@ export default class LessonDescriptions extends React.Component {
       });
     });
 
-    this.setState({
-      buttonText: 'Imported',
-      importedDescriptions,
-      mismatchedLessons
-    });
+    this.setState(
+      {
+        buttonText: 'Imported',
+        importedDescriptions,
+        mismatchedLessons
+      },
+      this.props.updateLessonDescriptions(this.updatedLessonDescriptions())
+    );
   };
 
   importDescriptions = () => {
```

Comment:
I see one instance of `defaultValue` later in this file: https://github.com/code-dot-org/code-dot-org/blob/cd8943a6b4521375b2666344180d7525a4c9f5cb/apps/src/lib/levelbuilder/script-editor/LessonDescriptions.jsx#L206-L210 is that going to be a problem?

Reply:
I think since this input does not have an onChange with it it does not need to updated. In general I found it just warns if you keep the defaultValue vs value.